packages/game-app/src/signup/components/Signup/Signup.tsx

```diff
@@ -64,22 +58,27 @@ const Signup: React.FC<Props> = () => {
     [signup, handleSubmit],
   );
 
+  useEffect(() => {
```

Comment:
maybe we can move this into the useRetrieveDevOpsMaturities checking if they are not yet loaded, so that we will nor reload them in case of already in the state